example/Components/AdvancedDeepNestedData.js

```diff
@@ -13,45 +13,90 @@ const {
 
 const Form = ({ revalidation: { form, onChange, errors = {}, onSubmit }, onSubmit: submitCb }) =>
   (
-    <div className='form'>
+    <form className='form' noValidate onSubmit={(e) => { e.preventDefault(); onSubmit(({valid, form}) => valid ? submitCb(form) : null ) }}>
       <div className='formGroup'>
-        <label>Name</label>
+        <label htmlFor='name'>Name</label>
         <input
+          id='name'
           type='text'
           value={form.name}
+          autoFocus
```

Comment:
`autofocus` attribute helps most users by focusing the first field in the form. This can be set dynamically (like first field with an error or first empty field if the user is continuing a session).